mergify_engine/merge_train.py

```diff
@@ -273,13 +277,25 @@ async def update_summaries(
             report,
         )
 
+        # NOTE(sileht): refresh it, so the queue action will merge it and delete the
+        # tmp_pull_ctxt branch
         async with utils.aredis_for_stream() as redis_stream:
             await github_events.send_refresh(
                 self.train.repository.installation.redis,
                 redis_stream,
                 original_ctxt.pull,
             )
 
+        if status in [check_api.Conclusion.SUCCESS, check_api.Conclusion.FAILURE]:
+            await tmp_pull_ctxt.client.post(
+                f"{tmp_pull_ctxt.base_url}/issues/{self.queue_pull_request_number}/comments",
+                json={"body": tmp_pull_title},
+            )
+            await tmp_pull_ctxt.client.patch(
+                f"{tmp_pull_ctxt.base_url}/pulls/{self.queue_pull_request_number}",
+                json={"state": "closed"},
+            )
```

Comment:
I'm a bit confused, I thought we already closed it anyway? Where is/was that code?

Reply:
Not yet. Actually, we don't close it, it's GitHub that closes it because we delete the branch. I just make it explicit.

Reply:
Oh ok!